utopia-remix/app/routes/projects.tsx

```diff
@@ -98,9 +109,10 @@ const ProjectsPage = React.memo(() => {
   useCleanupOperations()
 
   const data = useLoaderData() as unknown as {
```

Comment:
OT - note for a future PR - maybe we can extract all of this logic (till line 160) to a custom hook `useFilteredProjects()`, just for it to be more testable/readable

Reply:
yeah good shout 👍